programs/odbc-bridge/MainHandler.cpp

```diff
@@ -110,9 +110,12 @@ void ODBCHandler::handleRequest(HTTPServerRequest & request, HTTPServerResponse
 
     try
     {
-        auto connection_handler = ODBCConnectionFactory::instance().get(
-                validateODBCConnectionString(connection_string),
-                getContext()->getSettingsRef().odbc_bridge_connection_pool_size);
+        nanodbc::ConnectionHolderPtr connection_handler;
+        if (getContext()->getSettingsRef().odbc_bridge_use_connection_pooling)
+            connection_handler = ODBCConnectionFactory::instance().get(
```

Comment:
What do you think about renaming `ODBCConnectionFactory` to `ODBCPooledConnectionFactory` because you use it only for that case?
Just to make it a bit more clear.